### PR TITLE
Change the german translation

### PR DESCRIPTION
**Pull request by JoniRousu**
With all respect to the previous translator, I have found it not very useful since it is full of spelling mistakes and so called false-friend translations. To be honest it looked a bit like a draft. 

So I sat down and translated basically all of it again.

### CODE REVIEW

File: script/make-html.js [hunk 1]
```diff
@@ -22,13 +22,13 @@ const pageData = [
...
+        languageName: 'Deutsch',
         markdownFileName: 'content-de.md',
-        title: 'Eine interaktive Einführung in die Fourier-Transformationen',
-        description: 'Fourier Transformationen sind ein Werkzeug die für eine menge verschiedener anwendunge genutzt werden können. dies ist eine Erklärung was eine Fouriertransformation macht und einige andere Arten wie sie nützlich sein kann.',
+        title: 'Eine interaktive Einführung in die Fouriertransformation',
+        description: 'Die Fouriertransformation ist ein Werkzeug, das für eine Vielzahl verschiedener Anwendungen genutzt werden kann. Diese Seite ist eine Einführung in die Funktionsweise und Anwendung der Fouriertransformation.',
         outFileName: 'de.html',
         url: '/de.html',
-        translatorMarkdown: 'Übersetzt von [Frank H. Dürkopf](http://duerkopf.de)',
+        translatorMarkdown: 'Übersetzt von Joni Rousu',
```
Comment (Jezzamonn) [inline script/make-html.js]: We should still give credit to Frank here. Definitely add yours here, but include them both.

Or alternatively you can let me do it, but I'd be relying on Google translate...

**Comment by Jezzamonn** [comment]
Oh wow, thanks for the heads up, and thanks for correcting it! Left 1 comment to tweak, then I'll update the site

**Comment by JoniRousu** [comment]
@Jezzamonn, added the previous translator back as requested. I'm glad you like it!

File: content/content-de.md [hunk 2]
```diff
@@ -1,172 +1,173 @@
...
+Das funktioniert wirklich bei allen Zeichnungen! Dies ist deine Chance damit herumzuspielen.
 
 <div class="multi-container">
 <div class="sketch" >
     <canvas id="draw-zone" class="sketch-child" width=500 height=500></canvas>
-    <p id="draw-zone-instruction" class="instruction">Zeichne hier</p>
-    <button id="draw-zone-undo-button" class="button embedded-button">Zurücknehmen</button>
+    <p id="draw-zone-instruction" class="instruction">Zeichne hier!</p>
+    <button id="draw-zone-undo-button" class="button embedded-button">rückgängig</button>
 </div>
```
Comment (Jezzamonn) [inline content/content-de.md]: Should this be capitalised? The English is.
(I'll merge this now but if so let me know and I can make the change)

Comment (JoniRousu) [reply content/content-de.md]: Yeah, it should be capitalised. Personally I would prefer it in lowercase but it seems to be best practice to capitalise (e.g Gmail).

**Pull request merged**